app/webpacker/components/Users/List/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import {
+  Header, Input, Pagination, Table,
+} from 'semantic-ui-react';
+import { useQuery } from '@tanstack/react-query';
+import I18n from '../../../lib/i18n';
+import useDebounce from '../../../lib/hooks/useDebounce';
+import { getPersons } from '../api/getUsers';
+import Loading from '../../Requests/Loading';
+import WCAQueryClientProvider from '../../../lib/providers/WCAQueryClientProvider';
+import { RegionSelector } from '../../CompetitionsOverview/CompetitionsFilters';
+import { personUrl, editPersonUrl } from '../../../lib/requests/routes.js.erb';
+import { countries } from '../../../lib/wca-data.js.erb';
+
+export default function Wrapper() {
+  return (
+    <WCAQueryClientProvider>
+      <PersonList />
+    </WCAQueryClientProvider>
+  );
+}
+
+function PersonList() {
+  const [query, setQuery] = useState('');
+  const [page, setPage] = useState(1);
+  const [region, setRegion] = useState('all');
+
+  const debouncedSearch = useDebounce(query, 1000);
```

Comment:
We have a `WCA_DEFAULT_DEBOUNCE` or similar constant defined somewhere :tm: 
If you look for all usages of `useDebounce` (there really aren't that many) you'll quickly find it

Reply:
We actually do not. Everyone seems to be defining their own debounce time
![image](https://github.com/user-attachments/assets/3a7756fe-f729-4d81-b629-3c93dd82be94)

Reply:
we have window.wca.TEXT_INPUT_DEBOUNCE_MS for the old js code which is 250ms which I think is to low for this usecase

Reply:
Oh wow, that's funny :joy: Then maybe go with either 300 or 600 based on what we already have?

Reply:
600 feels decent